glyphs-reader/src/font.rs

```diff
@@ -41,6 +41,21 @@ pub struct Font {
     pub codepoints: BTreeMap<Vec<u32>, String>,
     // tag => (user:design) tuples
     pub axis_mappings: BTreeMap<String, RawUserToDesignMapping>,
+    pub features: Vec<FeatureSnippet>,
+}
+
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub enum FeatureType {
+    Prefix,
+    Class,
+    Feature,
+}
+
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub struct FeatureSnippet {
+    pub feature_type: FeatureType,
```

Comment:
why a separate FeatureType enum contained in the FeatureSnippet struct's feature_type field, and not instead a FeatureSnippet _enum_ where each variant carries with it the respective snippet name/code?

Reply:
It seemed the fields are the same which made such an enum feel weird to me. Maybe I can just not have the type field at all.